pulsar-functions/transforms/src/main/java/org/apache/pulsar/functions/transforms/TransformStep.java

```diff
@@ -0,0 +1,5 @@
+package org.apache.pulsar.functions.transforms;
+
+public interface TransformStep {
+    void process(TransformRecord transformRecord);
```

Comment:
I would add "throws Exception" or "throws ProcessingException" (where ProcessingException is a new checked Exception)

Reply:
done